Sources/NIOSSL/SSLErrors.swift

```diff
@@ -52,6 +52,7 @@ public typealias NIOBoringSSLErrorStack = [BoringSSLInternalError]
 /// Errors that can be raised by NIO's BoringSSL wrapper.
 public enum NIOSSLError: Error {
     case writeDuringTLSShutdown
+    @available(*, deprecated, message: "unableToAllocateBoringSSLObject is no longer available.")
```

Comment:
Can we reword this to say "can no longer be thrown"?

Reply:
Sounds good.  Would this be just "unableToAllocateBoringSSLObject can no longer be thrown," or just "can no longer be thrown?"

Reply:
Let's use the full language.

Also see the discussion in the main thread re: the equatable implementation.